client/src/main/java/org/eclipse/hono/client/Command.java

```diff
@@ -154,6 +152,15 @@ public Message getCommandMessage() {
         return message;
     }
 
+    /**
+     * Checks if this command is a <em>one-way</em> command (meaning there is no response expected).
+     *
+     * @return {@code true} if this is a valid command.
+     */
+    public boolean isOneWay() {
```

Comment:
do we need this?

Reply:
We will need it in the adapter implementation to decide if a command was one way (for convenience).
So, it is not strictly necessary in this commit, but in the directly following step 2 then, so I decided to provide it here already.

Reply:
Right.